account/partials/dashboard.njk

```diff
@@ -0,0 +1,47 @@
+{% extends "page.njk" %}
+{% from "components/button/macro.njk" import govukButton %}
+{%- from "components/table/macro.njk" import govukTable -%}
+
+{% block pageTitle %}Active Applications - {{ super() }}{% endblock %}
+
+{% block innerContent %}
+    <h1 class="govuk-heading-xl">Active Applications</h1>
+    <h2 class="govuk-heading-m">An application is <strong>saved for 30 days<strong> from when you start it.</h2>
```

Comment:
Should this be a heading, or just styled to be the size that it should?

Reply:
It was only done for the font size. I've made it a <p> and added a class to increase the font size to 24.